### ISSUE TITLE
SyntaxError when type '?' for a list of *magic* commands

### ISSUE THREAD

**Issue by kelsa-pi** [pyzo/pyzo#545]
HI,
I got an error message when type ?  for a list of *magic* commands.
```
>>> ?
  File "<console>", line 1
    ?
    ^
SyntaxError: invalid syntax
```
**Version info**
Pyzo version: 4.5.2 (source)
Platform: linux
Python version: 3.6.5
Qt version: 5.9.5
PyQt5 version: 5.10.1

Best

**Comment by almarklein** [comment]
Confirmed ...

**Comment by almarklein** [comment]
Fixed in e5606902cabe4c8d45ee8d5b67a0543756c3d614
